syncedlyrics/providers/musixmatch.py

```diff
@@ -60,11 +61,12 @@ def _get_token(self):
         with open(token_path, "w") as token_file:
             json.dump(token_data, token_file)
 
-    def get_lrc_by_id(self, track_id: str) -> Optional[str]:
+    def get_lrc_by_id(self, track_id: str) -> Optional[Lyrics]:
         r = self._get(
             "track.subtitle.get",
             [("track_id", track_id), ("subtitle_format", "lrc")],
         )
+        print(self.lang)
```

Comment:
I think this should be removed

Reply:
Your're absolutely right, I accidentally left it in there from testing